src/Microsoft.OpenApi/Models/OpenApiDocument.cs

```diff
@@ -29,7 +29,7 @@ public class OpenApiDocument : IOpenApiSerializable, IOpenApiExtensible
         /// <summary>
         /// REQUIRED. The available paths and operations for the API.
         /// </summary>
-        public OpenApiPaths Paths { get; set; } = new OpenApiPaths();
```

Comment:
> = new OpenApiPaths(); [](start = 47, length = 22)

The reason we had this was because OpenApiPaths is a collection (a dictionary). Is it causing issue?

Reply:
Yes.  We don't raise provide a validation error when a user provides a document that doesn't have a paths property.  We used to validate this in the reader, but we since moved it into our Validator.